packages/eslint-plugin/docs/rules/consistent-type-assertions.mdx

```diff
@@ -119,7 +119,7 @@ const foo = <Foo props={{ bar: 1 } as Bar} />;
 
 {/* insert option description */}
 
-Always prefer `const x: T[] = [ ... ];` to `const x = [ ... ] as T[];` (or similar with angle brackets).
+For example, this would prefer `const x: T[] = [ ... ];` to `const x = [ ... ] as T[];` (or similar with angle brackets).
```

Comment:
Matching the description with [`objectLiteralTypeAssertions`](https://github.com/typescript-eslint/typescript-eslint/blob/a94004c84d96b413ea92124cc3b120d5fa217adb/packages/eslint-plugin/docs/rules/consistent-type-assertions.mdx?plain=1#L49-L50), as the option description is very similar: `Whether to always prefer type declarations for array literals used as variable initializers, rather than type assertions.`